### PR TITLE
Fix add function

### PR DESCRIPTION
**Pull request by stevekinney**
This pull request seeks to add the correct way to add two numbers.

### CODE REVIEW

**Comment by martensonbj** [comment]
Way to catch that error. 👍

**Comment by martensonbj** [comment]
Oops. I owe you a $1.

File: script.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 function add(a, b) {
-  return a - b;
+  return b + a;
```
Comment (stevekinney) [inline script.js]: I should totally switch these arguments.

**Comment by stevekinney** [comment]
:shipit:

**Pull request merged**